seeds.sql

```diff
@@ -1,11 +1,2532 @@
-VALUES
-	(21, X'546F616420526561646572', X'3139322E3136382E312E37373A3139303036', X'61646D696E407265736F757263696E67656475636174696F6E2E636F6D', '2022-02-17 00:00:00.000', '2023-12-01 00:00:00.000', NULL, NULL, X'454D41494C', NULL, NULL, NULL, X'', NULL, NULL, NULL, X'6B426149466738494E676534464139794746765743744662426B7039325671754C3939524F56376A', NULL, NULL, NULL, X'68747470733A2F2F706C61792E676F6F676C652E636F6D2F73746F72652F617070732F64657461696C733F69643D636F6D2E746F61647265616465722E64656D6F', X'68747470733A2F2F6974756E65732E6170706C652E636F6D2F75732F6170702F746F61642D7265616465722F6964313431353039393436383F6D743D38', 0, NULL, NULL, NULL, NULL, 1, NULL, NULL, 40, 15, NULL, NULL, '2017-05-01 00:00:00.000', NULL, 1, NULL, NULL, X'68747470733A2F2F73332E616D617A6F6E6177732E636F6D2F63646E2E746F61647265616465722E636F6D2F74656E616E745F6173736574732F6C6F676F2D746F61647265616465722E706E67', 1);
+INSERT INTO idp (id, name, domain, contactEmail, use_enhanced_reader_at, use_audiobooks_at, specialPricing, fromEmail, authMethod, sessionSharingAsRecipientInfo, entryPoint, logoutUrl, nameQualifier, idpcert, spcert, spkey, internalJWT, userInfoEndpoint, userInfoJWT, actionEndpoint, androidAppURL, iosAppURL, xapiOn, xapiEndpoint, xapiUsername, xapiPassword, xapiMaxBatchSize, readingSessionsOn, consentText, amplitudeSecretKey, maxMBPerBook, maxMBPerFile, googleAnalyticsCode, language, created_at, demo_expires_at, deviceLoginLimit, emailBGColor, emailInnerBGColor, emailLogoUrl, emailHideName) VALUES
+(1, 'IDP Provider 1', 'idp1.example.com', 'admin1@example.com', '2024-06-15 12:41:45.680', '2024-10-11 12:41:45.680', 'OLD', 'noreply@idp1.example.com', 'NONE_OR_EMAIL', NULL, NULL, NULL, 'Qualifier-1', NULL, NULL, NULL, NULL, NULL, NULL, NULL, 'https://play.google.com/store/apps/details?id=idp1.app', 'https://itunes.apple.com/us/app/idp1', 0, NULL, NULL, NULL, NULL, 1, NULL, NULL, 24, 8, NULL, 'en', '2021-05-04 12:41:45.680', NULL, 8, '#FFFFFF', '#000000', 'https://s3.amazonaws.com/idp1/logo.png', 1),
+(2, 'IDP Provider 2', 'idp2.example.com', 'admin2@example.com', NULL, '2025-02-02 12:41:45.680', NULL, 'noreply@idp2.example.com', 'SESSION_SHARING', NULL, NULL, NULL, 'Qualifier-2', NULL, NULL, NULL, NULL, NULL, NULL, NULL, 'https://play.google.com/store/apps/details?id=idp2.app', 'https://itunes.apple.com/us/app/idp2', 0, NULL, NULL, NULL, 50, 1, NULL, NULL, 43, 12, NULL, 'es', '2015-12-29 12:41:45.680', '2025-04-03 12:41:45.680', NULL, '#F5F5F5', '#F5F5F5', 'https://s3.amazonaws.com/idp2/logo.png', 1),
```

Comment:
I think [idp's id 2](https://github.com/biblemesh/ereader-development/issues/3#issuecomment-2665517054) needs to point to the test Shibboleth instance for our testing to work